templates/macros/event.njk

```diff
@@ -1,8 +1,12 @@
-{% macro image(twitterInfo, width=350) %}
+{% macro image(twitterInfo, width=350, class) %}
```

Comment:
why not saying that the default value is `""` so then the `{% if class %}` is not needed? we could assign directly the value. (Or do some defensive checking inside in order not to add the conditional to the creation of the image)

btw, `class` is a (future) reserved word in js, I tend to use the name `className` for that concept, could you change the name?

Reply:
You're right, I thought about it and did it that way to avoid the empty class attribute, but it makes more sense to have a simpler logic!!